policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,47 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure development lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products.
+
+## Policy
+This lifecycle integrates security into all aspects of application software development which is comprized of six phases. 
+
+1. Requirements Gathering and Planning:
+  - Identify the scope, goals, and necessary resources for development
+  - Define functional and security requirements for the feature 
+    - Considering core engineering principles (link)
+    - Security controls such as authentication, data encryption, and secure communication protocols
+
+2. Design:
+- Create a feature description that incorporates identified security controls and meets definition of ready (link)
+- Perform threat modeling to identify potential risks and design appropriate countermeasures
+- Document design decisions, security controls, implementation and verification
+
+3. Development:
+- Adhere to coding standards and secure coding practices (link)
+- Implement features and security controls according to the design
+- Conduct code reviews to identify and address issues
+- Ensure automated code quality and security testing checks are passing prior to testing
+
+3. Testing:
+- Perform functional testing to validate the application meets desired requirements
+- Conduct security testing identified during the planning phase (e.g. static code analysis, dynamic scanning, and penetration testing)
+- Address identified issues and vulnerabilities, and validate the effectiveness of fixes
+- Document testing activities and results for the feature
+
+4. Deployment:
```

Comment:
Maybe consolidate 4-6 into a single phase

Reply:
It makes sense to me that these would be listed separately, but I could be swayed.

Reply:
I consolidated into 2 phases.